### PR TITLE
Removing term Dvcfile in push cmdref

### PR DESCRIPTION
**Pull request by imhardikj**
Partially address #1445 
* [x] push cmd ref.

### CODE REVIEW

File: content/docs/command-reference/push.md [hunk 1]
```diff
@@ -164,7 +164,6 @@ Posts.tsv.dvc
 Posts-test.tsv.dvc
 matrix-train.p.dvc
 model.p.dvc
-Dvcfile
```
Comment (imhardikj) [inline content/docs/command-reference/push.md:167]: @jorgeorpinel can we use example-get-started repo here in example.
And also update it with `dvc dag`.

**Comment by imhardikj** [comment]
@jorgeorpinel I have added `example-get-started` repo here in `push` example.

File: content/docs/command-reference/push.md [hunk 2]
```diff
@@ -157,37 +157,51 @@ a [pipeline](/doc/command-reference/pipeline) has been setup with these
...
-Posts-test.tsv.dvc
-matrix-train.p.dvc
-model.p.dvc
-Dvcfile
+$ dvc dag
+             +-------------+
+             | prepare.dvc |
+             +-------------+
+                    *
+                    *
```
Comment (imhardikj) [inline content/docs/command-reference/push.md:165]: @jorgeorpinel The [dag example](https://dvc.org/doc/command-reference/dag#examples) in cmd ref doesn't have `.dvc` in stage name it prints. Should I modify `dvc dag` ?

**Comment by jorgeorpinel** [comment]
Thanks @imhardikj, can you please reopen this when #1573 is merged? We don't have the capacity to review that many parallel PRs.

**Comment by jorgeorpinel** [comment]
> can you please reopen this when your other PR (#1573) is merged

And request my review, thanks

image

**Pull request closed without merging**